src/pages/SubmissionDetails.js

```diff
@@ -126,7 +112,7 @@ export const SubmissionDetails = ({ submissionId }) => {
       ]),
       div({ style: { marginLeft: '4em' } }, [
         h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`]),
-        h2(['workflow: ', getSpecificMethod[0]?.name]),
+        h2(['workflow: ', specifyRunSet[0]?.run_set_name]),
```

Comment:
Maybe move the workflow name down into the `h3` details, but we still want it. And this header should now probably be titled "Run Set: "